jenkins-pipelines/nonroot-offline-install/artifacts-centos8.jenkinsfile

```diff
@@ -8,7 +8,7 @@ artifactsPipeline(
     backend: 'gce',
     nonroot_offline_install: true,
     provision_type: 'spot',
-    scylla_mgmt_repo: '',
+    manager_branch: '',
```

Comment:
https://github.com/scylladb/scylla-cluster-tests/commit/954c10a9dbcf616dcb0142f53126a856841ff12f didn't elaborate much, I suppose because these are offline tests and the manager installation is online-dependent?